### DISCUSSION TITLE
Lack of GDPR Compliance

### DISCUSSION THREAD

**Discussion by transdeveloper** [yhype/yhype-feedback#37] [Q&A] [answered]
Hi,

I believe this service may not align with international data protection regulations, particularly GDPR. For instance, I couldn't find any option to delete my profile, which is frustrating as I now have to rely on Google to hide the results.

Additionally, it seems users are being signed up without prior consent, which I’m fairly certain is not compliant with legal standards.

Regardless, please remove me - along with whatever data you may have built up.

Warm regards,
Sophia

**Comment by antonkomarev** [comment, marked as answer]
If you still want to hide or delete your account you can do it by yourself in [Y HYPE settings page](https://yhype.me/settings)

<img width="1512" height="828" alt="image" src="https://github.com/user-attachments/assets/5c07ce7c-4f14-46b7-af94-f55d96a6d5f7" />

Or write me here. I've implemented admin panel for it.

**Comment by transdeveloper** [comment]
Hey @antonkomarev! Thanks for the update.

I've just deleted my account, and I still appear to be listed as under follower/following tabs under other accounts that got indexed.
Any reference to my username also needs to be deleted, however I don't think that's the case right now, and below you should see what I'm talking about.

<img width="790" height="378" alt="image" src="https://github.com/user-attachments/assets/b91e563e-9ff9-45ec-b1e6-8e65d7055f61" />

**Reply by antonkomarev** [reply]
@transdeveloper account deletes in one month, as defined in the request form.
